Controller/ProfileManager.php

```diff
@@ -6,8 +6,11 @@
 use PUGX\MultiUserBundle\Model\UserDiscriminator;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\RedirectResponse;
+use FOS\UserBundle\Model\UserInterface;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-class ProfileManager
+class ProfileManager extends Controller
```

Comment:
This class is not a controller

Reply:
This was the only way I managed to make it work when I did this PR. Maybe now there is another way